build.gradle

```diff
@@ -34,3 +34,57 @@ allprojects {
         ]
     }
 }
+
+// 0. clean up old files and register that task
+tasks.register('cleanUpChromeFolders', Delete) {
```

Comment:
Creating separate task for each tiny action seems excessive.
Can't we just create 2 tasks?
```
task.register("downloadChromium") {
  // cleanup
  // ensure dir exists
  // download
  // unzip
}

task.register("downloadChromeDriver") {
  // cleanup
  // ensure dir exists
  // download
  // unzip
}
```

Reply:
done